### PR TITLE
update padlock display behaviour

### PR DESCRIPTION
**Pull request by robmint**
Padlocks be like...

Screen Shot 2021-07-20 at 12 35 17 PM
Screen Shot 2021-07-20 at 12 35 11 PM

### CODE REVIEW

**Comment by robmint** [comment]
Apologies for use of auto-format feature, which obscures the real changes somewhat.

**Comment by rosemcc** [comment]
> Apologies for use of auto-format feature, which obscures the real changes somewhat.

I want to create a branch where I go through and autoformat EVERYTHING!!! Haha

File: research-hub-web/src/app/components/shared/body-media/body-media.component.html [hunk 1]
```diff
@@ -82,51 +93,77 @@
...
                     <a href="{{ contentItem.document ? contentItem.document.url : contentItem.url }}" target="_blank">
                         <mat-card class="embedded-entry-block">
-                            <mat-card-title><a href="{{ contentItem.document ? contentItem.document.url : contentItem.url }}" target="_blank">{{ contentItem.title }}</a><mat-icon *ngIf="contentItem.ssoProtected" matTooltip="SSO Login Required">lock</mat-icon></mat-card-title>             
-                            <mat-card-content><span class="card-summary">{{ contentItem.summary }}</span></mat-card-content> 
+                            <mat-card-title><a
+                                    href="{{ contentItem.document ? contentItem.document.url : contentItem.url }}"
+                                    target="_blank">
+                                    {{ contentItem.title }}
+                                </a>
+                                <mat-icon *ngIf="contentItem.ssoProtected" matTooltip="SSO Login Required">lock</mat-icon>
```
Comment (rosemcc) [inline research-hub-web/src/app/components/shared/body-media/body-media.component.html:125]: do we need &nbsp; here too?

**Review by rosemcc — CHANGES_REQUESTED** [review]
Looks good, just need to address the comment I left about the Link Card

**Review by rosemcc — APPROVED** [review]

**Pull request merged**